CHANGELOG.md

```diff
@@ -85,7 +85,10 @@ entirely to allow inference to fill in the type.
 ### Tool Changes
 
 * Pub
-    * Added support for the Dart Development Compiler in `build` and `serve`.
```

Comment:
Sorry...

Does this cleanup of the changelog include behavior changes from the CL?

If yes, those should be in the 1.25 section – this patch won't make 1.24